### ISSUE TITLE
Rawkit 0.6.0 does not support libraw 0.19.x

### ISSUE THREAD

**Issue by c-anna** [photoshell/rawkit#154]
The documentation page for rawkit says that it supports libraw 0.19.x, but it will fail to run with such a version of libraw installed. The code at `libraw/bindings.py:42` basically specifies the supported versions of libraw. At the `0.6.0` tag (the latest release), it is the following:

```python
        try:
            structs = {
                16: structs_16,
                17: structs_17,
                18: structs_18,
            }[self.version_number[1]]
        except KeyError:
            raise ImportError(
                'Unsupported Libraw version: %s.%s.%s.' % self.version_number
            )
```

If libraw 0.19.0 is installed, a KeyError will be raised here. However, the very latest code on `master` adds `19: structs_19`  to that dictionary, effectively supporting libraw 0.19.x.

When can a version of rawkit be released that supports libraw 0.19.x? And in the meantime, can the documentation be changed so as not to mislead users?

**Comment by SamWhited** [comment]
> The documentation page for rawkit says that it supports libraw 0.19.x, but it will fail to run with such a version of libraw installed

@c-anna can you provide the links to the docs you're reading? I suspect you're reading the docs for the latest master version (which has support for libraw 0.19.1+, but is slightly broken and thus hasn't been released) and not 0.6.0.

These are the docs for 0.6.0: https://rawkit.readthedocs.io/en/v0.6.0/ (which are actually missing 18)

> When can a version of rawkit be released that supports libraw 0.19.x?

> And in the meantime, can the documentation be changed so as not to mislead users?

The documentation is not broken, but it would make sense to set v0.6.0 (or whatever the latest actual release is) to the default. Unfortunately, Read The Docs doesn't appear to allow that (or if it does, I couldn't find a setting for it without changing the default source control branch, which isn't really the same thing and it makes sense for that to be master, even if the release version is older).

**Comment by c-anna** [comment]
Yes, I was reading the docs at https://rawkit.readthedocs.io/ (no version specified), which as you say was probably for the latest `master` version. I was just confused because I assumed that the docs I was viewing were tied to the "latest" release, which wasn't true then (but seems to be true now). Thank you!
